doc/source/ray-overview/workloads.md

```diff
@@ -7,7 +7,7 @@
 Explore the ways that Ray helps to build AI applications.
 
 ## Batch inference on CPUs and GPUs
-Performing inference on incoming batches of data can be parallelized by exporting the architecture and weights of a trained model to the shared object store. Using these model replicas, Ray scales predictions on batches across workers.
+Performing inference on incoming batches of data can be parallelized by exporting the architecture and weights of a trained model to the shared object store. Using these model replicas, Ray AIR's [Batch Predictor](../ray-air/predictors) scales predictions on batches across workers.
```

Comment:
Can we use a relative reference rather than using a relative path? It should be like `air-predictors`

Reply:
TIL! Thank you!